src/utils/fcc-test-suite-skeleton.html

```diff
@@ -24,6 +24,11 @@
     <div class="fcc_legend key"></div>
     <span class="fcc_legend">Tests Executing</span>
   </div>
+  <span id="fcc_report-bug"><a
+    href="https://www.freecodecamp.org/forum/c/help"
+    target="_blank">Stuck?</a>
```

Comment:
```suggestion
    target="_blank">Need Help?  Ask a question on our forum.</a>
```

Reply:
How about just `Need Help?`? The current suggestion would occupy a lot of space in the already small test box

Reply:
I like @RandellDawson's suggestion, but yes - it might take up a lot of space on that little box - I can't say if it would look fine or not without seeing it. How about `Ask for help` if it needs to be smaller

Reply:
`Ask For Help` sounds good. I'll commit that!